client/components/tinymce/plugins/simple-payments/dialog/product-image.jsx

```diff
@@ -17,14 +18,14 @@ import { url as mediaUrl } from 'lib/media/utils';
 import QueryMedia from 'components/data/query-media';
 import { getMediaItem } from 'state/selectors';
 
-const ProductImage = ( { siteId, imageId, image } ) => {
+const ProductImage = ( { siteId, imageId, image, showEditIcon } ) => {
 	if ( ! siteId || ! imageId ) {
 		return null;
 	}
 
 	if ( ! image ) {
 		return (
-			<figure className="editor-simple-payments-modal__figure is-placeholder">
+			<figure className="dialog__editor-simple-payments-modal-figure is-placeholder">
```

Comment:
Did we get a naming warning from eslint? Do we need to rename this one?

Reply:
Yes, I changed these and similar lines to avoid linting errors.